packages/core/upload/server/controllers/admin-folder-file.js

```diff
@@ -152,7 +168,7 @@ module.exports = {
             );
 
           // update path of files below
-          await strapi.db
+          totalFileNumber = await strapi.db
```

Comment:
Doesn't `.update()` return an array of updated entries?

Reply:
Only if we ask it to, by using `.returning()` or giving returning params in `.update()` :)